### PR TITLE
Start writing about #EmberJS2019 posts

### PR DESCRIPTION
**Pull request by amyrlam**
My thought is that quotes like 
image can be screenshotted with Skitch from the Netlify build, and be inserted as images in the Goodbits.

### CODE REVIEW

File: source/2019-06-01-the-ember-times-issue-101.md [hunk 1]
```diff
@@ -11,15 +11,37 @@ responsive: true
...
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [#EmberJS2019 🛣🗺](https://blog.emberjs.com/2019/05/20/ember-2019-roadmap-call-for-posts.html)
   
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+🚨 Blog posts wanted! 🚨 
+
+We would like you to write a blog post to propose **goals and direction** for Ember for the next year. The content of these posts will help the Core Team to draft their **next Roadmap RFC**. Please share your posts by **Monday, July 17th**.
```
Comment (ijlee2) [inline source/2019-06-01-the-ember-times-issue-101.md]: Can we also mention GitHub gists and tweets as alternatives to blog posts? I think the deadline is June 17th, to be considered as part of RFC.

File: source/2019-06-01-the-ember-times-issue-101.md [hunk 2]
```diff
@@ -11,15 +11,37 @@ responsive: true
...
+
+<a class="ember-button ember-button--centered" href="https://blog.emberjs.com/2019/05/20/ember-2019-roadmap-call-for-posts.html">Read more about the Roadmap RFC</a>
+
+Looking to see what's already out there? Posts are being gathered at [this community repo](https://github.com/abhilashlr/emberjs2019-posts) - thanks [@abhilashlr](https://github.com/abhilashlr)! We also wanted to share some highlights from posts out there in the Times. 
+
+---
+
+[@ijlee2](https://github.com/ijlee2) wants Ember to [Build a Larger Community](https://crunchingnumbers.live/2019/05/26/emberjs2019-build-a-larger-community/).
+
+> Our community, while [truly amazing and supportive](https://blog.emberjs.com/2019/04/12/the-ember-times-issue-93.html), is yet small. To flourish, we need **support from developers who don’t work with Ember daily**. These developers may professionally work with React, Angular, or Vue. They may be self-taught or attending school, looking to enter tech with minimal risk in career trajectory and minimal time to create showcase projects.
```
Comment (ijlee2) [inline source/2019-06-01-the-ember-times-issue-101.md:34]: :blushed:

**Review by ijlee2 — APPROVED** [review]
Thank you!

File: source/2019-06-01-the-ember-times-issue-101.md (hunk 1 above)
Comment (amyrlam) [reply source/2019-06-01-the-ember-times-issue-101.md]: fixed both - geez, that would have been a bad error saying July! 😅

**Comment by amyrlam** [comment]
let's get this in, can edit at will on the main branch / future PR's

**Pull request merged**